api-report/routerlicious-driver.api.md

```diff
@@ -62,6 +62,14 @@ export interface ITokenService {
     extractClaims(token: string): ITokenClaims;
 }
 
+// @public (undocumented)
+export enum R11sErrorType {
```

Comment:
In addition to my naming comment, we should add documentation to this type and its flags, now that we're exporting it.